### PR TITLE
Get the storybook version from metadata

### PR DESCRIPTION
**Pull request by kasperpeulen**

### CODE REVIEW

File: services/test-results.ts [hunk 1]
```diff
@@ -36,16 +36,30 @@ function getTestResultFromDay(enrichedDay: ReturnType<typeof enrichDayWithData>,
...
 
+function getMetadata(pipeline: EnrichedPipeline, template: string): { version: string } | undefined {
+  const buildJob = pipeline.jobs.find((it) => it.name === 'build-sandboxes');
+  if (buildJob == null) return;
+  const metaDataTest = buildJob.tests.find((it) => it.name.includes(`${template} - metadata`));
+  if (metaDataTest == null) return;
+  try {
+    return JSON.parse(metaDataTest.message);
+  } catch (e) {
+    return;
```
Comment (yannbf) [inline services/test-results.ts:59]: I assume we want to throw the error? but just in case it's actually an invalid JSON

**Comment by yannbf** [comment]
BTW there's a bug in the status page story, can you take a look at it?

File: services/test-results.ts (hunk 1 above)
Comment (kasperpeulen) [reply services/test-results.ts]: I actually just gonna bubble up this error, as we really should not get it.

**Comment by kasperpeulen** [comment]
@yannbf The status page was broken because I accidentally pushed the mocks that I generated from the PR (to test the metadata).

**Pull request merged**